shifthelper/tools/__init__.py

```diff
@@ -26,13 +26,19 @@
 db_engines = {}
 
 
-@retry(stop_max_delay=30000,  # 30 seconds max
-       wait_exponential_multiplier=100,  # wait 2^i * 100 ms, on the i-th retry
-       wait_exponential_max=1000,  # but wait 1 second per try maximum
-       )
 def get_alerts():
```

Comment:
This means we never get called if the website is down. I think the developer should get called if the web service is not reachable for a longer time.

Reply:
oh

Reply:
Or we say: If something happens, the shifter is called, realises the website is down and calls the expert. That woould minimize the number of calls the developer receives in the  night.